backend/app/expenses/service.py

```
@@ -0,0 +1,1095 @@
+            if updates.tags is not None:
+                update_doc["tags"] = updates.tags
+            if updates.receiptUrls is not None:
+                update_doc["receiptUrls"] = updates.receiptUrls
+
+            # Only add history if there are actual changes
+            if len(update_doc) > 1:  # More than just updatedAt
+                # Get user name
+                try:
+                    user = await self.users_collection.find_one({"_id": ObjectId(user_id)})
+                    user_name = user.get("name", "Unknown User") if user else "Unknown User"
+                except:
```

Flake8 (7.2.0)

[error] do not use bare 'except'

(E722)